source/_layouts/guide.liquid

```diff
@@ -0,0 +1,481 @@
+---
```

Comment:
@jgerigmeyer @mirisuzanne I took a pass at switching over to using a liquid template for the learn sass page / code helper. The indentation works for me locally now and in the code its much easier to read too. I also switched over to a capture block for the code example. Let me know what you think about this approach!